RELEASE-NOTES.md

```diff
@@ -1,5 +1,67 @@
 # Prefect Release Notes
 
+## Release 2.16.5
+
+### Exciting New Features 🎉
+- Render artifacts and states on the flow run graph by default. — https://github.com/PrefectHQ/prefect/pull/12300
+
+### Enhancements
+- Keep artifacts file in prefect-client — https://github.com/PrefectHQ/prefect/pull/12316
+- Add a select all flow runs checkbox to the flow runs page. — https://github.com/PrefectHQ/prefect/pull/12356
+- remove feature flagging around enhanced-deployment-experiment — https://github.com/PrefectHQ/prefect/pull/12360
```

Comment:
@jakekaplan is this worth highlighting or do you know if this was already highlighted before?

Reply:
@serinamarie I think this was highlighted before because we defaulted it to `True` and made it opt-out?

Reply:
Confirmed with @pleek91 it was highlighted 2 weeks ago

Reply:
niiiice!